Content.Shared/Shipyard/Prototypes/VesselPrototype.cs

```diff
@@ -39,6 +40,12 @@ public sealed class VesselPrototype : IPrototype
     [DataField("group")]
     public string Group = string.Empty;
 
+    /// <summary>
+    ///     The access of the product. (e.g. Command, Mail, Bailiff, etc.)
```

Comment:
Frontier mention

Reply:
Looking into this this all file should be moved under _NF but that a different issue.